src/debugAdapter/mdbCommunications.ts

```diff
@@ -199,6 +203,36 @@ export class MDBCommunications extends EventEmitter {
 		return result;
 	}
 
+	/**
+	 * Handles a "Stop at" message from output. Swallows responses until entirety of "Stop at" message has been chunked out
+	 * @param initialMessage The initial message containing "Stop at" and potentially more of the data
+	 */
+	private async handleStopAt(initialMessage: string): Promise<void> {
+		let message = initialMessage;
+		let matches = message.match(/address:(0x.{8})|file:(.+)|source line:(\d+)/gm);
```

Comment:
`/address:(0x.{8})|file:(.+)|source line:(\d+)/gm` is used two times in this function. Please make a const that is used in both places. Something like: `const breakPointRegex = /address:(0x.{8})|file:(.+)|source line:(\d+)/gm`

Reply:
Good point, changed